query-engine/driver-adapters/src/proxy.rs

```diff
@@ -243,8 +256,9 @@ fn js_value_to_quaint(
         ColumnType::DateTime => match json_value {
             serde_json::Value::String(s) => {
                 let datetime = chrono::NaiveDateTime::parse_from_str(&s, "%Y-%m-%d %H:%M:%S%.f")
+                    .map(|dt| DateTime::from_utc(dt, Utc))
+                    .or_else(|_| DateTime::parse_from_rfc3339(&s).map(DateTime::<Utc>::from))
```

Comment:
TODO: this is sufficient for now to make smoke tests work but quaint supports more formats, do we need those?